parsec/backend/app.py

```diff
@@ -247,7 +247,12 @@ async def handle_client(self, stream):
                 pass
 
     async def handle_client_http(self, stream, event, conn):
-        status_code, headers, data = get_method_and_execute(event.target, self.backend_addr)
+        if hasattr(self, "running_backend_addr"):
+            status_code, headers, data = get_method_and_execute(
+                url=event.target, backend_addr=self.running_backend_addr
+            )
+        else:
+            status_code, headers, data = get_method_and_execute(url=event.target, backend_addr=None)
```

Comment:
Nice fix in theory, but if that was the correct attribute, shouldn't we provide a "guessed" `backend_addr` instead if it hasn't been defined by the user? (If I remember correctly from what @touilleMan told me months ago, `backend_addr` is used in case the local config of the server is different from what is accessible from the outside)

Also, where is supposed to be defined `self.backend_addr` in the nominal case? Arent't we just supposed to use `self.config.backend_addr` instead, for which existence is guaranteed?

Reply:
```status_code, headers, data = get_method_and_execute(
    event.target, self.config.backend_addr if self.config.backend_addr else "https://localhost"+self.config.port
)```